modules/oxide/src/less/theme/components/form/color.less

```diff
@@ -16,7 +16,7 @@
     }
 
     span {
-      border-color: rgba(red(@color-black), green(@color-black), blue(@color-black), .2);
+      border-color: rgba(red(@color-black) green(@color-black) blue(@color-black) / 20%);
```

Comment:
I don't know if I'll ever get used to the new syntax. This still looks to me like it's dividing the blue by 20% (I know it's not).